demos/CompositeUI-MVC/Divergent.Frontend/App_Start/ContainerConfig.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Castle.MicroKernel.Registration;
+using Castle.MicroKernel.Resolvers.SpecializedResolvers;
+using Castle.Windsor;
+using Divergent.ITOps.ViewModelComposition;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Web;
+using System.Web.Mvc;
+using System.Web.Optimization;
+
+namespace Divergent.Frontend
+{
+    public class ContainerConfig
+    {
+        public static IWindsorContainer ConfigContainer()
```

Comment:
It feels like the class should be called `ContainerFactory` and this method should be called `Create()`.